apps/emqx_dashboard/src/emqx_dashboard_schema.erl

```diff
@@ -76,6 +76,42 @@ fields("listeners") ->
             )}
     ];
 fields("http") ->
+    [
+        {"enabled",
```

Comment:
In other places we commonly use the configuration name `enable` instead of `enabled`. I'm not quite sure which one would be better, but it should be better to be consistent?

Reply:
`"enabled"` is preferred. IMO.

Reply:
sorry, "enable"